src/mono/nuget/Microsoft.NET.Workload.Mono.Toolchain.Current.Manifest/WorkloadManifest.json.in

```diff
@@ -197,7 +197,10 @@
     },
     "Microsoft.NET.Runtime.WebAssembly.Templates.${NetVersion}": {
       "kind": "template",
-      "version": "${PackageVersion}"
+      "version": "${PackageVersion}",
+      "alias-to": {
+        "any": "Microsoft.NET.Runtime.WebAssembly.Templates"
```

Comment:
The true packageid includes .NET versionhttps://github.com/dotnet/runtime/pull/107182

Reply:
ahh ok. I got mislead because https://www.nuget.org/packages/Microsoft.NET.Runtime.WebAssembly.Templates does exist, but there's https://www.nuget.org/packages/Microsoft.NET.Runtime.WebAssembly.Templates.net9 too